### PR TITLE
Add instructions to mount and inspect encrypted backups

### PR DESCRIPTION
**Pull request by gStart9**

### CODE REVIEW

**Comment by gStart9** [comment]
Do we need more explicit instructions for copying files from the backups to eg. `/embassy-data/package-data/volumes/*/*`?

File: site/source/user-manual/backups/backup-mount.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+==============
+Mount a Backup
+==============
+
+.. contents::
+  :depth: 2 
+  :local:
+
+In order to inspect the contents of an encrypted Start9 server backup, you first need to create a backup to mount and inspect.
```
Comment (kn0wmad) [inline site/source/user-manual/backups/backup-mount.rst:11]:
```suggestion
    .. warning:: The following are advanced tools and using them is only advised when under the instruction of a Start9 Support Technician.  USE AT YOUR OWN RISK - these operations are not supported outside of the context of support.
```

**Review by kn0wmad — CHANGES_REQUESTED** [review]
Sorry to butcher this one up, but I want to convey that this is not for normie use

**Comment by kn0wmad** [comment]
> Do we need more explicit instructions for copying files from the backups to eg. `/embassy-data/package-data/volumes/*/*`?

No, just basic documentation is fine, thanks

**Review by kn0wmad — APPROVED** [review]

**Pull request merged**